src/dtree.ml

```diff
@@ -0,0 +1,304 @@
+open Terms
```

Comment:
Thanks to add a reference to Luc's paper (bibliographic details + DOI URL).

Reply:
See 970de3c